[2022-12-08 16:28:45,467] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_redshift_to_s3 manual__2022-12-08T16:28:42.021660+00:00 [queued]>
[2022-12-08 16:28:45,474] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_redshift_to_s3 manual__2022-12-08T16:28:42.021660+00:00 [queued]>
[2022-12-08 16:28:45,474] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-08 16:28:45,474] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-12-08 16:28:45,474] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-08 16:28:45,483] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): move_data_from_redshift_to_s3> on 2022-12-08 16:28:42.021660+00:00
[2022-12-08 16:28:45,488] {standard_task_runner.py:52} INFO - Started process 744 to run task
[2022-12-08 16:28:45,491] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'move_data_from_s3_to_redshidt_with_awswrangler', 'move_data_from_redshift_to_s3', 'manual__2022-12-08T16:28:42.021660+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/move_data_s3redshift.py', '--cfg-path', '/tmp/tmpzs1e9ngx', '--error-file', '/tmp/tmpe1sppl9b']
[2022-12-08 16:28:45,492] {standard_task_runner.py:80} INFO - Job 102: Subtask move_data_from_redshift_to_s3
[2022-12-08 16:28:45,528] {task_command.py:369} INFO - Running <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_redshift_to_s3 manual__2022-12-08T16:28:42.021660+00:00 [running]> on host 8458a2c1ba2e
[2022-12-08 16:28:45,583] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=move_data_from_s3_to_redshidt_with_awswrangler
AIRFLOW_CTX_TASK_ID=move_data_from_redshift_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2022-12-08T16:28:42.021660+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-08T16:28:42.021660+00:00
[2022-12-08 16:28:45,595] {credentials.py:1120} INFO - Found credentials in environment variables.
[2022-12-08 16:28:45,966] {credentials.py:1120} INFO - Found credentials in environment variables.
[2022-12-08 16:28:46,753] {python.py:173} INFO - Done. Returned value was: None
[2022-12-08 16:28:46,763] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=move_data_from_s3_to_redshidt_with_awswrangler, task_id=move_data_from_redshift_to_s3, execution_date=20221208T162842, start_date=20221208T162845, end_date=20221208T162846
[2022-12-08 16:28:46,784] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-08 16:28:46,808] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
